README.md

```diff
@@ -887,6 +887,8 @@ Most `docker_container` properties are the `snake_case` version of the
 - `host` - A string containing the host the API should communicate with.
   Defaults to local `docker_service`.
 - `host_name` - The hostname for the container.
+- `labels` A hash or array to set metadata on the container in the form
```

Comment:
docker_service says that it requires Set, is it the same?

Probably dash is missing before `A`